flow-typed/npm/source-map_v0.7.x.js

```diff
@@ -1,12 +1,12 @@
-// flow-typed signature: 5dc4832be8a4e3b97e16fecc54691aac
```

Comment:
Do we really have to update this? The new mappings this PR introduces aren't correct?

Reply:
Agreed. I'm happy to wait to land this until we can upstream some changes to the libdef.

Did we modify the prior one or do you know if there were bugs introduced with this newer libdef?

Reply:
We modified the prior one

I apparently never opened a PR to `flow-typed` will make one today

Reply:
It got merged 🎉